elpis/wrappers/utilities/json_utilities.py

```diff
@@ -4,26 +4,32 @@
 Copyright: University of Queensland, 2019
 Contributors:
              Nicholas Lambourne - (University of Queensland, 2018)
+             Ben Foley - (University of Queensland, 2020)
 """
 
 import json
-from typing import Union
+from json.decoder import JSONDecodeError
+import os
+from typing import List, Dict, Union
 from _io import TextIOWrapper
 
 
-def load_json_file(file_name: str) -> object:
+def load_json_file(file_name: str) -> List[Dict[str, str]]:
     """
     Given a filename (parameter) containing JSON, load and
     return the a list of python dictionaries with containing the same information.
     :param file_name: name of file containing JSON to read from.
     :return a Python dictionary with the contents of the JSON file.
     """
-    file = open(file_name, "r", encoding="utf-8")
-    data: object = json.load(file)
+    data = []
+    with open(file_name, "r", encoding="utf-8") as file_:
+        try:
+            data = json.load(file_)
+        except JSONDecodeError:
```

Comment:
Do you want JSON decoding to fail silently? What's the intention here?

Reply:
The intention is to return an empty list if the json file doesn't exist. I'll change it to check for `if file exists` instead.